tests/download/test_download_instance.py

```diff
@@ -6,21 +6,21 @@
 from vrplib import download_instance
 
 
-def test_deprecation_warning():
+def test_deprecation_warning(tmp_path):
     """
     Checks if a deprecation warning is raised when the function is called.
     """
     with pytest.warns(DeprecationWarning):
-        download_instance("X-n101-k25", "tmp")
+        download_instance("X-n101-k25", tmp_path / "tmp")
```

Comment:
Before I changed it, these tests would dump a `tmp` file in the project root. That's a bit annoying :).